test/e2e/e2e_server_test.go

```diff
@@ -39,7 +39,7 @@ func TestLogRequestId(t *testing.T) {
 	}
 	w := httptest.NewRecorder()
 	buff.Reset()
-	req, _ := http.NewRequest(http.MethodPost, "/", bytes.NewBufferString("hello"))
+	req, _ := http.NewRequest(http.MethodPost, "/proxy", bytes.NewBufferString("hello"))
```

Comment:
Is there a reason why this test was changed to use the `/proxy` endpoint?

Reply:
You are right. My views while replacement was to test everything with `/proxy` so that after this change once we deprecate we have testing in place. However I think it makes sense to just add test for `/proxy` separately and let other test do test on `/` itself.
Once we deprecate we will do the required change from test perspective. WDYT?

Reply:
makes sense - I think we can lgtm once this change here is reverted.

Reply:
Fixed in [commit](https://github.com/redhat-appstudio/sprayproxy/pull/30/commits/a88c888c0b71a89f78eaef416038f87330d21ba4)